test/runner/e2e_update_rel_test.cpp

```diff
@@ -0,0 +1,165 @@
+#include "test_helper/test_helper.h"
+
+using namespace kuzu::testing;
+
+class UpdateRelTest : public DBTest {
+public:
+    string getInputCSVDir() override {
+        return TestHelper::appendKuzuRootPath("dataset/rel-insertion-tests/");
+    }
+    string getUpdateRelQuery(string srcTable, string dstTable, string relation, int64_t srcID,
+        int64_t dstID, string setPropertyClause) {
+        return StringUtils::string_format(
+            "MATCH (p1:%s)-[e:%s]->(p2:%s) WHERE p1.ID = %d AND p2.ID = %d " + setPropertyClause,
+            srcTable.c_str(), relation.c_str(), dstTable.c_str(), srcID, dstID);
+    }
+    string getInsertKnowsRelQuery(string srcTable, string dstTable, int64_t srcID, int64_t dstID) {
```

Comment:
It is confusing that you're using srcID also as length property. Can you just pass a length argument here explicitly?

Reply:
Added a new parameter for length property.